src/cgame/cg_rocket.cpp

```diff
@@ -642,3 +639,29 @@ bool CG_Rocket_IsCommandAllowed( rocketElementType_t type )
 			return false;
 	}
 }
+
+bool CG_Rocket_LoadCursor( Str::StringRef cursorPath )
+{
+	if ( cursorPath.empty() )
+	{
+		rocketInfo.cursor = 0;
+		return true;
+	}
+	rocketInfo.cursor = trap_R_RegisterShader( cursorPath.c_str(), (RegisterShaderFlags_t) RSF_DEFAULT );
+	if ( rocketInfo.cursor == 0 )
+	{
+		return false;
+	}
+	// Scale cursor with resolution while maintaining the original aspect ratio.
+	int x, y;
+	trap_R_GetTextureSize( rocketInfo.cursor, &x, &y );
+	float ratio = static_cast<float>( x ) / static_cast<float>( y );
+	rocketInfo.cursor_pos.h = cgs.glconfig.vidHeight * 0.025f;
```

Comment:
`rocketInfo.cursor_pos.h = cgs.glconfig.vidHeight / 40.f;` would be more readable, to me. Ideally, would have liked some info about that divide, too, but not sure the info is still around?

Reply:
I guess. The factor/div is not that important, the most important thing would have been to know why the value, but since knowledge is lost, it's can't be helped.